File: app/src/main/java/com/rdev/tryp/model/favorite_address/Address.java
package com.rdev.tryp.model.favorite_address;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Address {

    @SerializedName("address")
    @Expose
    private String address;
    @SerializedName("lat")
    @Expose
    private Integer lat;
    @SerializedName("lng")
    @Expose
    private Double lng;

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public Integer getLat() {
        return lat;
    }

    public void setLat(Integer lat) {
        this.lat = lat;
    }

    public Double getLng() {
        return lng;
    }

    public void setLng(Double lng) {
        this.lng = lng;
    }

}

File: app/src/main/java/com/rdev/tryp/model/favorite_address/Address.kt
package com.rdev.tryp.model.favorite_address

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class Address {

    @SerializedName("address")
    @Expose
    var address: String? = null
    @SerializedName("lat")
    @Expose
    var lat: Int? = null
    @SerializedName("lng")
    @Expose
    var lng: Double? = null

}